python/sglang/srt/models/torch_native_llama.py

```diff
@@ -84,7 +95,7 @@ def __init__(
             intermediate_size * 2,
             bias=False,
         )
-        self.gate_up_proj.output_sizes = [intermediate_size] * 2
+        self.gate_up_proj.output_sizes = [intermediate_size // tp_size] * 2
```

Comment:
so we still have model changes?

Reply:
I can also move the `// tp_size` entirely into the weight loader if we prefer keeping the model side clean.

Reply:
yeah that might be better I feel

Reply:
Done, thanks!